packages/vhd-lib/merge.js

```diff
@@ -132,6 +132,13 @@ class Merger {
 
     // merging vhdFile must not be concurrently with the potential block reordering after a change
     this.#mergeBlockConcurrency = parentIsVhdDirectory && childIsVhdDirectory ? this.#mergeBlockConcurrency : 1
+
+    if (parentIsVhdDirectory && !isVhdAlias(this.#parentPath)) {
+      const error = new Error("can't merge vhd directories without using alias")
+      error.code = 'NOT_SUPPORTED'
+      throw error
+    }
+
```

Comment:
Is this related to this PR?

In the unit tests, it seems you are testing against another error.

Reply:
Yes, maybe I should have split this into two commits : one for directory+alias , and one with the cache message 

are you thinking of this line https://github.com/vatesfr/xen-orchestra/pull/6597/files#diff-e60c4dfeb47bccfac6bae88c7a85172b9ef22e9f1bb87a7a8d36232ca00a0b65R387or is there another ?

Reply:
Yes, I'm thinking of this line.

Reply:
I may be tired, but I think it's the same errror code, no  ?

Reply:
Yes but not the same message unless I'm mistaken.

Reply:
the third parameter of strictequal is a message when the test, it's not used for comparison

Reply:
My bad!